components/cam/src/control/ncdio_atm.F90

```diff
@@ -569,8 +569,9 @@ subroutine infld_real_2d_3d(varname, ncid, dimname1, dimname2,              &
       !
       ! Get array dimension id's and sizes
       !
-      ierr = PIO_inq_dimid(ncid, dimname1, arraydimids(1))
-      ierr = PIO_inq_dimid(ncid, dimname2, arraydimids(2))
+      ! Only do this check if the dimension name does not include '_d'
```

Comment:
What does this code do?

Reply:
If I remember this correctly, this is checking for a dimension named "ncol" in the file. "ncol" always refers to the physics grid, so when using the FV physgrid the dynamics grid version of that dimension is named "ncol_d". In order to maintain the standard configuration we need to work around the fact that the file being read here will always have an "ncol" dimension instead of "ncol_d". So the simple solution is to bypass this check. We can do something fancier in the future if/when the FV phsygrid gets adopted as the default. The important thing is that it does not affect the standard configuration, since it will still use that check.